src/Microsoft.AspNet.Http.Abstractions/HttpRequestStreamReader.cs

```diff
@@ -0,0 +1,438 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Buffers;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http.Abstractions;
+
+namespace Microsoft.AspNet.Http
```

Comment:
This shouldn't be in the `Http.Abstractions` package, it should be in web utilities in this repo.